RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2795,6 +2795,19 @@ boolean providePlusNonCombat(int amt, boolean doEquips)
 		}
 	}
 
+	// We can get these during normal game, may as well use them!
+	if(0 >= have_effect($effect[Fresh Scent]))
```

Comment:
this briefly confused me, I suggest using `==`

Reply:
Done, thanks! :)

Bad habits from interfacing bash with tcl. a null comes across as -1